### PR TITLE
Optionally append lang for packaged model name

### PR DESCRIPTION
**Pull request by svlandeg**
## Description
* Only prepend the language code if it's not there already - otherwise calling `package` with for instance `name="nl_core_my_pipeline"` would result in an `"nl_nl_core_my_pipeline"` package.
* Nitpicking: add empty newline at the end of the Python files that are written.

### Types of change
enhancement

## Checklist
- [x] I have submitted the spaCy Contributor Agreement.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: spacy/cli/package.py [hunk 1]
```diff
@@ -112,7 +112,9 @@ def package(
         msg.fail("Invalid pipeline meta.json")
         print("\n".join(errors))
         sys.exit(1)
-    model_name = meta["lang"] + "_" + meta["name"]
+    model_name = meta["name"]
+    if not model_name.startswith(meta['lang']):
```
Comment (adrianeboyd) [inline spacy/cli/package.py:116]: Maybe `model_name.startswith(meta['lang'] + "_")` instead?

Overall I'm a little worried this change is going to cause too much unexpected behavior.

Comment (svlandeg) [reply spacy/cli/package.py:116]: You think? I think people who were using it correctly (not giving the language code with the name) won't see a difference, right? You think there's people actually aiming for a name like "nl_nl_core_my_pipeline"? :|

Comment (adrianeboyd) [reply spacy/cli/package.py:116]: No, I think you might want `nl_nlcorpusname_something`.

Comment (svlandeg) [reply spacy/cli/package.py:116]: Yea, potentially. I actually had the check first like you suggested so I'll put it back like that.

Comment (svlandeg) [inline spacy/cli/package.py:116]:
```suggestion
    if not model_name.startswith(meta['lang'] + "_"):
```

Comment (svlandeg) [reply spacy/cli/package.py]: Do you still think it's potential breaking/unexpected then?

I actually found the "nl_nl_" bit unexpected myself :p

Comment (adrianeboyd) [reply spacy/cli/package.py]: I guess I'm not worried that it's unexpected, just that it's a change that could break things for existing workflows.

Comment (svlandeg) [reply spacy/cli/package.py]: There's always a chance, true :| 
We could put it behind an additional flag, a boolean whether to prepend lang, that is by default `True`? 
Or whatever, I'm not invested in this feature either 😁

Comment (adrianeboyd) [reply spacy/cli/package.py]: That seems like overkill, and I think the number of potentially affected users is really really small.

**Comment by svlandeg** [comment]
We should hold off on merging until we're ready to publish 3.1

**Pull request merged**